src/components/Pagination/Pagination.scss

```diff
@@ -1,47 +1,212 @@
 @import "./../../base/base.scss";
 
-.nitrozen-pagination-container {
+.n-pagination-container {
   font-family: $PrimaryFont;
   color: $TypographyPrimaryColor;
-  padding: 1.2rem;
+  border: 1px solid $ColorPrimaryGrey60;
+  border-radius: 12px;
+  padding: 12px 12px;
 
-  .nitrozen-pagination {
+  .n-pagination {
     display: flex;
     font-size: 1.4rem;
     color: $LabelColor;
     font-size: 1.4rem;
     font-weight: 400;
-
-    .nitrozen-pagination__left {
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 12px;
+    .mobile_view {
+      display: none !important;
+      @media only screen and (max-width: $sm-min) {
+        display: flex !important;
+      }
+    }
+    .n-pagination__left {
       flex: 1;
       display: flex;
       align-items: center;
+      color: rgba(0, 0, 0, 0.65);
+      @media only screen and (max-width: $sm-min) {
+        display: none;
+      }
+      .n-pagination__count {
+        color: rgba(0, 0, 0, 0.65);
+        font-weight: 500;
+        font-size: 14px;
+        line-height: 20px;
+        @media only screen and (max-width: $sm-min) {
+          font-size: 12px;
+        }
+      }
+    }
+    .n-pagination__main {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      gap: 36px;
+      flex-wrap: wrap;
+      @media only screen and (max-width: $sm-min) {
+        gap: 18px !important;
+      }
+      .n-pagination__number {
+        display: flex;
+        gap: 8px;
+        flex-direction: row;
+        position: relative;
+        flex-wrap: wrap;
+        @media only screen and (max-width: $sm-min) {
+          gap: 4px !important;
+        }
+      }
+      .n-pagination__showpopup {
+        position: absolute;
+        padding: 12px;
+        z-index: 1;
+        background: #ffffff;
+        box-shadow: 0px 4px 16px rgb(0 0 0 / 8%);
+        border-radius: 16px;
+        width: 184px;
+        height: auto;
+        display: flex;
+        flex-direction: column;
+        gap: 8px;
+        top: 50px;
+
+        .n-pagination__search_input {
+          border: 1px solid $ColorPrimaryGrey60;
+          border-radius: 12px;
+          padding: 10px;
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          gap: 8px;
+          input::placeholder {
+            font-weight: 500;
+            font-size: 14px;
+            line-height: 20px;
+            letter-spacing: -0.005em;
+            color: rgba(0, 0, 0, 0.65);
```

Comment:
Please use standard color values from variables instead of hardcoding rgba values. It'll cause an issue in maintainability of styles